create-projects/tests/run.sh

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env bash
+set -ue
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PATH=${SCRIPT_DIR}:$PATH
+
+curl --silent -L "https://raw.githubusercontent.com/michaelsauter/bock/master/bock.sh" -o "${SCRIPT_DIR}/oc" && chmod +x "${SCRIPT_DIR}/oc"
```

Comment:
Could it make sense to not use master but a fixed 'release' of bock.sh?

Reply:
Yes - done in 3eda1d7.